server/util.go

```diff
@@ -249,3 +251,31 @@ func InitHTTPClient(svr *Server) error {
 	}}
 	return nil
 }
+
+const matchRule = "^[A-Za-z0-9]([A-Za-z0-9_-]*[A-Za-z0-9])?$"
+
+// ValidateLabelString checks the legality of the label string.
+// the valid label should start as alphanumeric and can have an underscore or hyphen after,
```

Comment:
The first letter should be capitalized. And maybe `start with an alphanumeric character` is better?

Reply:
BTW, the regular expression says it can start with numbers too.